.github/workflows/cs-lint.yml

```diff
@@ -62,9 +62,9 @@ jobs:
           xml-schema-file: ./vendor/phpunit/phpunit/phpunit.xsd
 
       # Check the code-style consistency of the PHP files.
-#      - name: Check PHP code style
-#        continue-on-error: true
-#        run: vendor/bin/phpcs --report-full --report-checkstyle=./phpcs-report.xml
+      - name: Check PHP code style
+        continue-on-error: true
```

Comment:
As this solves all CS violations, then I would remove this line, so that any changes (either from adjusted code, or fresher versions of PHPCS-related dependencies) get highlighted.
```suggestion
```